README.md

```diff
@@ -36,167 +40,46 @@ Refer to these files by adding a `<link>` and a `<script>` element into your HTM
-include both JavaScript and Sass files.
-
-Running `require( 'uswds' )` will include the contents of the `components.js`
-file in your project. This file does not export currently export anything so it
-doesn't need to be assigned to a variable. This library depends on `jQuery` and
-`$` to be on the `window` object before it is required. So please be sure that
-you include the jQuery dependency on your HTML page or via `npm` and set it to
-the `window` object as `jQuery` and `$`.
-
-```js
-window.$ = window.jQuery = require( 'jquery' );
-require( 'uswds' );
-// ...
-```
-
-To use the styles from the Draft Web Design Standards you can import it into
-your main Sass file like so:
-
-```scss
-@import './node_modules/uswds/assets/_scss/all.scss';
-```
-
-Your HTML file may be structured like this:
-
-```html
-<!-- CSS Includes -->
-<link rel="stylesheet" href="./node_modules/uswds/assets/css/google-fonts.css">
-
-<!-- Your compiled CSS, which imports the Draft Web Design Standards -->
-<link rel="stylesheet" href="./path/to/assets/style.css">
-
-<!-- Your compiled JavaScript, which creates jQuery on the `window` object. -->
-<script src="./path/to/assets/start-app.js"></script>
-```
-
-Note: You might get an [`npm` warning related to `lodash`](https://github.com/18F/web-design-standards/pull/902#issuecomment-161076213), but you can generally ignore it.
-This error is related to a dependency found in `node-sass`.
+And that’s it — you should be set to use the Standards.
 
-## Setup for your local environment
+## Using NPM
 
-### Requirements
+If you have `node` installed on your machine, you can use npm to install the Standards. Add `uswds` to your `package.json` as a dependency or a development dependency:
 
-These setup instructions are for running the [Draft Web Design Standards website](https://playbook.cio.gov/designstandards/) locally on your machine and not for using the assets (CSS, JavaScript, etc) on your own project. If you want to use the standards assets, go to [Getting Started](https://github.com/18F/web-design-standards#getting-started). For context, the styleguide website uses [Jekyll](http://jekyllrb.com/) (a static site generator), which is run via a [`./go` script](https://github.com/18F/go_script). At 18F, we then use [18F Pages](https://github.com/18F/pages) to publish and serve the site.
+`npm install --save[-dev] uswds`
 
-You will need [Ruby](https://www.ruby-lang.org) ( > version 2.2.3 ). You may consider using a Ruby version manager such as [rbenv](https://github.com/sstephenson/rbenv) or [rvm](https://rvm.io/) to help ensure that Ruby version upgrades don't mean all your [gems](https://rubygems.org/) will need to be rebuilt.
+The package will be installed in `node_modules` under `uswds`, and the Sass stylesheets will be available for you to compile using your project’s asset pipeline. (You can also access a compiled version in the `dist/css/uswds.css`.)
```

Comment:
Also @msecret — would the minified version need a separate section w/its own intro? Or could this info be seamlessly included in the current section?